deployment/terraform/storage.tf

```diff
@@ -0,0 +1,18 @@
+resource "aws_s3_bucket" "logs" {
+  bucket = "${lower(var.project)}-${lower(var.environment)}-logs-${var.aws_region}"
+  acl    = "private"
+
+  tags {
+    Name        = "${lower(var.project)}-${lower(var.environment)}-logs-${var.aws_region}"
+    Project     = "${var.project}"
+    Environment = "${var.environment}"
+  }
+}
+
+module "ecr-repository" {
```

Comment:
I think that this resource name should use an underscore vs. a hyphen to be consistent with all of the others.